src/NzbDrone.Core/MediaFiles/DiskScanService.cs

```diff
@@ -131,6 +144,25 @@ public void Scan(List<string> folders = null, FilterFilesType filter = FilterFil
                 {
                     _logger.Debug("Specified scan folder ({0}) doesn't exist.", folder);
 
+                    if (_configService.CreateEmptyAuthorFolders)
+                    {
+                        if (_configService.DeleteEmptyFolders)
+                        {
+                            _logger.Debug("Not creating missing author folder: {0} because delete empty series folders is enabled", folder);
```

Comment:
series => author

Reply:
Oops, so right!
Do I fix it? What's next? New to this...